web/styles/left_sidebar.css

```diff
@@ -413,6 +413,10 @@ li.top_left_recent_topics {
     line-height: 1.25;
 }
 
+.pm-box[data-is-group="false"] .conversation-partners {
+    display: flex;
+}
```

Comment:
Would it be possible to avoid this fairly subtle CSS rule with a slight refactor of the template rendering logic, for example to have each individual be in its own span, to better control the overflow/line-wrapping without a conditional on `data-is-group`?

Reply:
Possibly. I would have to investigate how the ellipsis overflow would act in relation to multiple inline elements (versus just a straight text node, which is I believe what `text-overflow: ellipsis` is designed to handle).

My broader reservation is in introducing still more DOM nodes (potentially hundreds or thousands), especially since in this case they would have the sole purpose of giving slightly more styling control--but not much else.

One option to get away from the `data-is-group` selector (which I agree is suboptimal) would be to rewrite or hook into the template logic, and just drop a plain old class with the same meaning as `data-is-group="false"`. That might even be something like using a singular `.conversation-partner` class in the singleton case.

But from doing this work, I also get a strong feeling that how usernames, status emoji, avatars, and presence indicators are presented all need some deeper thinking to make them more uniform, less brittle, and better citizens of the DOM to try and reduce the number of nodes wherever possible.